test/buildConnectorsJson/prettyTitle.js

```diff
@@ -5,46 +5,55 @@ var prettyTitle = require('../../lib/buildConnectorsJson/prettyTitle');
 
 describe('#prettyTitle', function () {
 
-	it('should make things look nice', function () {
-		assert.equal(prettyTitle('list_name'), 'List name');
-		assert.equal(prettyTitle('list-name'), 'List name');
-		assert.equal(prettyTitle('listName'), 'List name');
-		assert.equal(prettyTitle('ListName'), 'List name');
-		assert.equal(prettyTitle('listName'), 'List name');
+	it('should make names look nice', function () {
+		assert.strictEqual(prettyTitle('list_name'), 'List name');
+		assert.strictEqual(prettyTitle('list-name'), 'List name');
+		assert.strictEqual(prettyTitle('listName'), 'List name');
+		assert.strictEqual(prettyTitle('ListName'), 'List name');
+		assert.strictEqual(prettyTitle('listName'), 'List name');
 	});
 
 	it('should upper case id', function () {
-		assert.equal(prettyTitle('id'), 'ID');
-		assert.equal(prettyTitle('list_id'), 'List ID');
-		assert.equal(prettyTitle('list_id_name'), 'List ID name');
+		assert.strictEqual(prettyTitle('id'), 'ID');
+		assert.strictEqual(prettyTitle('list_id'), 'List ID');
+		assert.strictEqual(prettyTitle('list_id_name'), 'List ID name');
 
-		assert.equal(prettyTitle('lid'), 'Lid');
-		assert.equal(prettyTitle('list baseid'), 'List baseid');
+		assert.strictEqual(prettyTitle('lid'), 'Lid');
+		assert.strictEqual(prettyTitle('list baseid'), 'List baseid');
 	});
 
 	it('should upper case ids', function () {
-		assert.equal(prettyTitle('ids'), 'IDs');
-		assert.equal(prettyTitle('list_ids'), 'List IDs');
-		assert.equal(prettyTitle('list_ids_name'), 'List IDs name');
+		assert.strictEqual(prettyTitle('ids'), 'IDs');
+		assert.strictEqual(prettyTitle('list_ids'), 'List IDs');
+		assert.strictEqual(prettyTitle('list_ids_name'), 'List IDs name');
 
-		assert.equal(prettyTitle('lids'), 'Lids');
-		assert.equal(prettyTitle('list baseids'), 'List baseids');
+		assert.strictEqual(prettyTitle('lids'), 'Lids');
+		assert.strictEqual(prettyTitle('list baseids'), 'List baseids');
 	});
 
 	it('should upper case url', function () {
-		assert.equal(prettyTitle('url'), 'URL');
-		assert.equal(prettyTitle('list_url'), 'List URL');
-		assert.equal(prettyTitle('list_url_name'), 'List URL name');
+		assert.strictEqual(prettyTitle('url'), 'URL');
+		assert.strictEqual(prettyTitle('list_url'), 'List URL');
+		assert.strictEqual(prettyTitle('list_url_name'), 'List URL name');
 
-		assert.equal(prettyTitle('lurl'), 'Lurl');
-		assert.equal(prettyTitle('list baseurl'), 'List baseurl');
+		assert.strictEqual(prettyTitle('lurl'), 'Lurl');
+		assert.strictEqual(prettyTitle('list baseurl'), 'List baseurl');
+	});
+
+	it('should upper case ddl', function () {
+		assert.strictEqual(prettyTitle('ddl'), 'DDL');
+		assert.strictEqual(prettyTitle('list_ddl'), 'List DDL');
+
+		assert.strictEqual(prettyTitle('lurl'), 'Lurl');
+		assert.strictEqual(prettyTitle('list baseurl'), 'List baseurl');
```

Comment:
I don't think these should be in that test. Looks like your copied too much from the above test  😀

Reply:
good catch